jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -472,9 +494,27 @@ public Field run() {
  * @see			java.lang.Class
  */
 @CallerSensitive
-public static Class<?> forName(String className, boolean initializeBoolean, ClassLoader classLoader)
-	throws ClassNotFoundException
+public static Class<?> forName(
+	String className, boolean initializeBoolean, ClassLoader classLoader) throws ClassNotFoundException
+{
+/*[IF JAVA_SPEC_VERSION >= 18]*/
```

Comment:
the macro spanning multiple methods is a little confusing to read. Can you modify it such that the macro is entire contained in a method,  a method is entirely contained in the macro block

Reply:
Updated the code for the two specified locations. Added a new commit. I will squash after you confirm that the changes are good.